app/src/main/java/org/sopt/dosopttemplate/presentation/auth/LoginActivity.kt

```diff
@@ -7,24 +7,25 @@ import android.os.Bundle
 import android.view.MotionEvent
 import android.view.View
 import androidx.appcompat.app.AppCompatActivity
-import org.sopt.dosopttemplate.R
+import androidx.lifecycle.ViewModelProvider
 import org.sopt.dosopttemplate.data.User
 import org.sopt.dosopttemplate.databinding.ActivityLoginBinding
 import org.sopt.dosopttemplate.di.UserSharedPreferences
 import org.sopt.dosopttemplate.presentation.main.BnvActivity
 import org.sopt.dosopttemplate.util.BackPressedUtil
 import org.sopt.dosopttemplate.util.hideKeyboard
-import org.sopt.dosopttemplate.util.showShortSnackBar
-import org.sopt.dosopttemplate.util.showShortToast
 
 class LoginActivity : AppCompatActivity() {
     private lateinit var binding: ActivityLoginBinding
+    private lateinit var loginViewModel: LoginViewModel
 
     override fun onCreate(savedInstanceState: Bundle?) {
         binding = ActivityLoginBinding.inflate(layoutInflater)
         super.onCreate(savedInstanceState)
         setContentView(binding.root)
 
+        loginViewModel = ViewModelProvider(this).get(LoginViewModel::class.java)
```

Comment:
```suggestion
   
    private val loginViewModel by viewModels<LoginViewModel>()

    override fun onCreate(savedInstanceState: Bundle?) {
        binding = ActivityLoginBinding.inflate(layoutInflater)
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
```

build.gradle 추가된 아래 의존성 때문에 viewModel 생성함수를 좀 더 편하게 사용할 수 있습니다.
```
implementation "androidx.fragment:fragment-ktx:1.5.3"
implementation "androidx.activity:activity-ktx:1.6.0"
```

Reply:
오 지연초기화 때문에 두줄이었던 코드가 한줄로 바로 줄어드네여 쓰기 좋을 것 같습니다